From 913d9cc3db5415fbe210e709169064dd7bbeebd8 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Thu, 23 Aug 2018 11:13:48 +0800
Subject: [PATCH 4440/5242] MLK-19305-3 arm64: enable gpio PAD wakeup for
 i.MX8 platforms

commit  3770cad321e3321698a3dae034a3f4fe00fec214 from
https://source.codeaurora.org/external/imx/linux-imx.git

Select gpio PAD wakeup for those i.MX8 platforms with
system controller present.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
(cherry picked from commit 61f3cc34747e5c8db1578c8aba5bd9592533cede)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/Kconfig.platforms |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index f4ff97b..228b5ae 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -303,6 +303,7 @@ config ARCH_FSL_IMX8QM
 	select PINCTRL_IMX8QM
 	select PM
 	select SOC_BUS
+	select GPIO_MXC_PAD_WAKEUP
 	help
 	  This enables support for Freescale i.MX8QM SOC.
 
@@ -313,6 +314,7 @@ config ARCH_FSL_IMX8QP
 	select PINCTRL_IMX8QM
 	select PM
 	select SOC_BUS
+	select GPIO_MXC_PAD_WAKEUP
 	help
 	  This enables support for Freescale i.MX8QP SOC.
 
@@ -324,6 +326,7 @@ config ARCH_FSL_IMX8QXP
 	select PINCTRL_IMX8QXP
 	select PM
 	select SOC_BUS
+	select GPIO_MXC_PAD_WAKEUP
 	help
 	  This enables support for Freescale i.MX8QXP SOC.
 
-- 
1.7.9.5

